schemas/io.growthbook/experiment_viewed/jsonschema/1-0-0

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a GrowthBook experiment viewed event. See https://docs.growthbook.io/",
+  "self": {
+    "vendor": "io.growthbook",
+    "name": "experiment_viewed",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "experimentId": {
+      "type": "string",
+      "maxLength": 160,
+      "description": "The experiment tracking key or experiment id that uniquely identifies this experiment"
+    },
+    "variationId": {
+      "type": "string",
+      "maxLength": 100,
+      "description": "The experiment variation id that the user was exposed to"
+    },
+    "hashAttribute": {
+      "type": "string",
```

Comment:
Can we allow `null` values for optional fields? The optional but doesn't allow null type has caused us issues in the past.

```suggestion
      "type": [
		"null",
		"string"
	],
```

Reply:
Sure, I'll make this change.